packages/utils/src/misc.ts

```diff
@@ -378,19 +378,11 @@ export const crossPlatformPerformance: Pick<Performance, 'now' | 'timeOrigin'> =
     // is not as widely supported. Namely, performance.timeOrigin is undefined in Safari as of writing.
     // tslint:disable-next-line:strict-type-predicates
     if (performance.timeOrigin === undefined) {
-      // For webworkers it could mean we don't have performance.timing then we fallback
-      // tslint:disable-next-line:deprecation
-      if (!performance.timing) {
-        return performanceFallback;
-      }
-      // tslint:disable-next-line:deprecation
-      if (!performance.timing.navigationStart) {
-        return performanceFallback;
-      }
-
+      // As of writing, performance.timing is not available in Web Workers in mainstream browsers, so it is not always a
+      // valid fallback. In the absence of a initial time provided by the browser, fallback to INITIAL_TIME.
       // @ts-ignore
       // tslint:disable-next-line:deprecation
-      performance.timeOrigin = performance.timing.navigationStart;
+      performance.timeOrigin = performance.timing?.navigationStart || INITIAL_TIME;
```

Comment:
Careful with `performance.timing?` `?` 
our SDK doesn't use latest typescript so CI will fail for sure

Reply:
Updated.